graphite/graphite.go

```diff
@@ -26,12 +26,30 @@ type Series struct {
 
 type DataPoint []json.Number
 
+type Error struct {
```

Comment:
I'd prefer something like:

```
type Error struct {
  URL url.URL
  StatusCode int
  Body []byte // response body
}
```

All the fields are exportable and thus usable from timeGraphiteRequest.